packages/backup-restore-operator/charts/templates/backup.yaml

```diff
@@ -0,0 +1,92 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: backups.resources.cattle.io
+spec:
```

Comment:
Are we including any [`additionalPrinterColumns`](https://kubernetes.io/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions/) or is that a later change?

Reply:
that will come later